src/libs/Middleware/Reauthentication.js

```diff
@@ -113,6 +111,17 @@ function Reauthentication(response, request, isFromSequentialQueue) {
 
             // Return response data so we can chain the response with the following middlewares.
             return data;
+        })
+        .catch((error) => {
+            // If the request is on the sequential queue, re-throw the error so we can decide to retry or not
+            if (isFromSequentialQueue) {
+                throw error;
+            }
+
+            // If we have caught a networking error from a deprecated api request, resolve it as unable to retry, otherwise the request will never resolve or reject.
```

Comment:
```suggestion
            // If we have caught a networking error from a DeprecatedAPI request, resolve it as unable to retry, otherwise the request will never resolve or reject.
```

Small suggestion as I think it will just help whoever has to clean this up remember to delete this code

Reply:
Oh yeah that's a good idea to make it easy to find in a search.